docs/operation/password-recovery.rst

```diff
@@ -1,18 +1,20 @@
+:lastproofread: 2025-11-19
+
 .. _password-recovery:
 
 #################
 Password Recovery
 #################
 
 Using the console, restart the VyOS router. The GRUB menu appears.
-Select the relevant option from the GRUB menu and press Enter.

```

Comment:
Is this password recovery flow still accurate?

Reply:
I think we should fix the boot menu so that the old process is relevant again. It's not a good idea to hide options that people may need in an emergency under a needless layer of indirection.